py2many/cli.py

```diff
@@ -366,15 +360,89 @@ def _process_once(settings, filename, outdir, env=None):
 
         if restore_cwd:
             os.chdir(restore_cwd)
+    except Exception as e:
+        print(f"Error: Could not format: {output_path}")
+        print(f"Due to: {e.__class__.__name__} {e}")
+        return False
 
     return True
 
 
+def _get_output_path(filename, ext, outdir):
+    output_path = outdir / (filename.stem + ext)
+    if ext == ".kt" and output_path.is_absolute():
+        # KtLint does not support absolute path in globs
+        output_path = _relative_to_cwd(output_path)
+    return output_path
+
+
+def _process_once(settings, filename, outdir, env=None):
```

Comment:
likewise putting this between `_get_output_path` and `_format_one` will result in a smaller diff